khal/calendar_display.py

```diff
@@ -49,7 +51,50 @@ def getweeknumber(date):
     return datetime.date.isocalendar(date)[1]
 
 
-def str_week(week, today):
+def get_event_color(event, default_color):
+    """Because multi-line lambdas would be un-Pythonic
+    """
+    if event.color == '':
+        return default_color
+    return event.color
+
+
+def str_highlight_day(day, devents, hmethod, default_color, multiple, color):
```

Comment:
I'd just return the color here, and not take a string. `style(..., fg=...)` is short enough. This eliminates the `hmethod` parameter.